api/src/api/services/phases/phase-processor.integration.test.ts

```diff
@@ -214,10 +223,13 @@ describe('PhaseProcessor Integration Test', () => {
       const processor = new PhaseProcessor();
       const rampService = new RampService();
       const quoteService = new QuoteService();
+
+      registerPhaseHandlers();
+
       const additionalData = {
         walletAddress: EVM_TESTING_ADDRESS,
         paymentData: {
-          amount: '1', // Relevant for test???
+          amount: '0.0000000001', // TODO this is user controlled, not only in test, perhaps we should protect. It should come from the quote.
```

Comment:
I think this is too low to be a valid Stellar amount. Stellar only supports 1e7 decimals.

Reply:
No, if we can remove it, that's better 👍